tests/functional/monitoring/prometheus/alerts/test_alert_mds_cache_high_usage.py

```diff
@@ -0,0 +1,234 @@
+import time
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import tier2
+from ocs_ci.ocs import constants
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import cluster
+from ocs_ci.utility import prometheus
+from ocs_ci.framework.pytest_customization.marks import blue_squad
+from ocs_ci.utility.utils import ceph_health_check_base
+from ocs_ci.ocs.node import (
+    unschedule_nodes,
+    drain_nodes,
+    schedule_nodes,
+)
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="function")
+def run_metadata_io_with_cephfs(pvc_factory, dc_pod_factory):
+    """
+    This function facilitates
+    1. Create PVC with Cephfs, access mode RWX
+    2. Create dc pod with Fedora image
+    3. Copy helper_scripts/meta_data_io.py to Fedora dc pod
+    4. Run meta_data_io.py on fedora pod
+    """
+    access_mode = constants.ACCESS_MODE_RWX
+    file = constants.METAIO
+    interface = constants.CEPHFILESYSTEM
+    log.info("Checking for Ceph Health OK")
+    ceph_health_check_base()
+
+    for i in range(3):
+        # Creating PVC with cephfs as interface
+        log.info(f"Creating {interface} based PVC")
+        pvc_obj = pvc_factory(interface=interface, access_mode=access_mode, size="30")
+        # Creating a Fedora dc pod
+        log.info("Creating fedora dc pod")
+        pod_obj = dc_pod_factory(
+            pvc=pvc_obj, access_mode=access_mode, interface=interface
+        )
+        # Copy meta_data_io.py to fedora pod
+        log.info("Copying meta_data_io.py to fedora pod ")
+        cmd = f"oc cp {file} {pvc_obj.namespace}/{pod_obj.name}:/"
+        helpers.run_cmd(cmd=cmd)
```

Comment:
since you have the pod_obj, you could use copy_to_pod_rsync func to copy the file from a local path to the pod path

Reply:
To approach that way, it will be adding few more lines of code and need to import Pod class for that. Here only once we copy the file to the pod. So I feel ok to use the single line command for now. If any issues arise then will look into this?